CHANGELOG.md

```diff
@@ -912,3 +912,4 @@ Initial version to have a stable reference before [gossip protocol removal](/doc
 - Bucket commands.
 - Downsampling support for UI.
 - Grafana dashboards for Thanos components.
+- [#3259](https://github.com/thanos-io/thanos/pull/3259) Thanos BlockViewer.
```

Comment:
Could you be a little more descriptive for the end user on what this PR adds? You can take inspiration from the PR title.

Reply:
Also this needs to be added at the top of the file under the "Unreleased" block.

Reply:
Thank you, I made the corrections

Reply:
LGTM 😊